server/routes/components.ts

```diff
@@ -19,6 +19,39 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
     return res.send(components)
   })
 
+  get('/veracode', async (req, res) => {
+    return res.render('pages/veracode')
+  })
+
+  get('/veracode/data', async (req, res) => {
+    const components = await serviceCatalogueService.getComponents()
+
+    const rows = components.map(component => {
+      const hasVeracode = !!component.attributes.veracode_results_summary
+      const severityLevels = {
+        LOW: 0,
+        MEDIUM: 0,
+        HIGH: 0,
+      }
+
+      component.attributes.veracode_results_summary?.severity?.forEach(severity => {
+        severity.category.forEach(category => {
+          severityLevels[category.severity] += category.count
+        })
+      })
+
+      return {
+        name: component.attributes.name,
+        hasVeracode,
+        pass: component.attributes.veracode_policy_rules_status === 'Pass',
```

Comment:
is this one mapped from the compliance status?

Reply:
Yep, I also discovered the way that types file gets generated means it stops at a particular level so I ended up having to amend it to give the entire severity bit as it just puts unknown :/

Reply:
should be mapped from policy_rules_status